docs/concepts/sys-update.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 9
+sidebar_label: "System Updates"
+title: About System Updates
+description: Update Treasury functionality including contract, policy, and app widget.
+---
+
+## About updates
+
+The [system update](../settings.md#system-updates) functionality allows Treasury developers to provide updates such as contract upgrades, policy changes, or instance app widget changes to treasury instances.
```

Comment:
I think for user it will easier if we say something like

```suggestion
The [system update](../settings.md#system-updates) allows Treasury developers to share important updates with treasury instances. These updates can relate to contract or frontend UI (e.g., Sputnik contract upgrades, DAO policy updates, or new UI features).